### ISSUE TITLE
TensorFlowTransformer: Passing constant value to the model

### ISSUE THREAD

**Issue by zeahmed** [dotnet/machinelearning#2413]
There should be a way to pass constant values to the TensorFlow model because there are cases when model requires an input which is constant regardless of the input feature vector.

For example, the [Batch Normalization](https://www.tensorflow.org/api_docs/python/tf/layers/batch_normalization) in TensorFlow requires a paramter called `training` which is set to `True` during training and `False` during inference. This variable is often set through as a `Placeholder` variable which is taken as input at runtime. 

When using such a model for inference in `TensorFlowTransformer`, we need to always pass `False` for this variable or always set it to `True` when training.
